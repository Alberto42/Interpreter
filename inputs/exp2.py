a = True or False
b = True and False
c = not False
d = 1 < 1
e = 1 <= 1
f = 1 > 1
g = 1 >= 1
h = 1 == 1
i = 1 != 1