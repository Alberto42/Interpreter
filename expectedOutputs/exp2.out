Ident "i" = VBoolean False
Ident "h" = VBoolean True
Ident "g" = VBoolean True
Ident "f" = VBoolean False
Ident "e" = VBoolean True
Ident "d" = VBoolean False
Ident "c" = VBoolean True
Ident "b" = VBoolean False
Ident "a" = VBoolean True


